agent/hcp/deps.go

```diff
@@ -109,3 +123,19 @@ func verifyCCMRegistration(ctx context.Context, client hcpclient.Client) (*url.U
 
 	return u, nil
 }
+
+// defaultLabels returns a set of string labels to be sent with each metrics Sink export
+// to the HCP Telemetry Gateway.
+func defaultLabels(cfgLabels map[string]string, nodeID types.NodeID) map[string]string {
+	// Use configured labels, if any.
+	labels := cfgLabels
+	if len(labels) == 0 {
+		labels = make(map[string]string, 2)
+	}
+
+	// Set default labels
+	labels["__replica__"] = string(nodeID)
+	labels["node_id"] = string(nodeID)
+
+	return labels
+}
```

Comment:
```suggestion
// getDefaultLabels returns a set of string labels to be sent with each metrics Sink export
// to the HCP Telemetry Gateway.
func getDefaultLabels(cfgLabels map[string]string, nodeID types.NodeID) map[string]string {
	labels := map[string]string{
		"__replica__": string(nodeID), // used for Cortex HA-metrics (deduplication)
		"node_id":     string(nodeID), // used to delineate Consul nodes in graphs
	}

	for k, v := range cfgLabels {
		labels[k] = v
	}

	return labels
}
```

diff spelling, same thing

Reply:
Thank you! Much prettier 🙏🏽